Vagrantfile

```diff
@@ -52,16 +52,20 @@ def vm_cpus
 end
 
 Vagrant.configure("2") do |config|
-  # always use Vagrants insecure key
+  # always use Vagrant's insecure key
   config.ssh.insert_key = false
-
   config.vm.box = "coreos-%s" % $update_channel
-  config.vm.box_version = ">= 681.2.0"
-  config.vm.box_url = "http://%s.release.core-os.net/amd64-usr/current/coreos_production_vagrant.json" % $update_channel
+  config.vm.box_version = "= 647.2.0"
```

Comment:
> This does work for coreos-stable, even if you delete that version.

Funny, it didn't work for me, and *shouldn't* work, as far as I can tell - see https://github.com/coreos/coreos-vagrant/issues/83#issuecomment-119814476

Without these changes, the client tries to fetch the metadata for 647.2.0 from http://stable.release.core-os.net/amd64-usr/current/coreos_production_vagrant.json, but the only version listed there is the latest in the channel. So, it errs out.

As for the issue with different channels, I'm not too concerned about supporting the update channel in the Vagrantfile workflow, as that is a seldom-used special case.